examples/db-connect.php

```diff
@@ -1,37 +1,46 @@
 <?php
+declare( strict_types = 1 );
 
-$dotenv = Dotenv\Dotenv::createImmutable(dirname(__DIR__));
+use Dotenv\Dotenv;
+
+$dotenv = Dotenv\Dotenv::createImmutable( dirname( __DIR__ ) );
 $dotenv->load();
 
 use Illuminate\Database\Capsule\Manager as Capsule;
 
 $capsule = new Capsule;
-$capsule->addConnection([
-    'driver'    => 'mysql',
-    'host'      => 'localhost',
-    'database'  => getenv('DATABASE'),
-    'username'  => getenv('USERNAME'),
-    'password'  => getenv('PASSWORD'),
-    'charset'   => 'utf8',
-    'collation' => 'utf8_unicode_ci',
-    'prefix'    => '',
-]);
+$capsule->addConnection(
+	[
+		'driver' => 'mysql',
+		'host' => 'localhost',
+		'database' => getenv( 'DATABASE' ),
+		'username' => getenv( 'USERNAME' ),
+		'password' => getenv( 'PASSWORD' ),
+		'charset' => 'utf8',
+		'collation' => 'utf8_unicode_ci',
+		'prefix' => '',
+	]
+);
 
 // Make this Capsule instance available globally via static methods... (optional)
 $capsule->setAsGlobal();
 
-function savePush(APNSPush $push, $token) {
+// phpcs:ignore NeutronStandard.Globals.DisallowGlobalFunctions.GlobalFunctions,WordPress.NamingConventions.ValidFunctionName.FunctionNameInvalid
+function savePush( APNSPush $push, $token ) {
 	global $capsule;
 
-	$capsule->table('mobile_push_queue')->insert([
-		'token'					=> $token,
-		'payload'				=> json_encode($push),
-		'when'					=> date("Y-m-d H:i:s"),
-		'mobile_push_client_id'	=> 0,
-	]);
+	$capsule->table( 'mobile_push_queue' )->insert(
+		[
+			'token' => $token,
+			'payload' => json_encode( $push ),
+			'when' => gmdate( 'Y-m-d H:i:s' ),
+			'mobile_push_client_id' => 0,
+		]
+	);
 }
 
-function getPushes($count = 10) {
+// phpcs:ignore NeutronStandard.Globals.DisallowGlobalFunctions.GlobalFunctions,WordPress.NamingConventions.ValidFunctionName.FunctionNameInvalid
```

Comment:
Same as above. Decided to ignore instead of fixing because this is just an example.